### ISSUE TITLE
How to generate segmentation labels for tusimple data？

### ISSUE THREAD

**Issue by daigang896** [voldemortX/pytorch-auto-drive#41]
Hello，
    How to generate segmentation labels for tusimple data？Because of the self generated label training, the accuracy ACC is very low, only 0.665.  Use：python main_landec.py --epochs=50 --lr=0.2 --batch-size=20 --dataset=tusimple --method=scnn --backbone=erfnet --mixed-precision --exp-name=erfnet_scnn_tusimple. Only segmentation labels are different. Is there a problem? Hope to provide some help?

**Comment by voldemortX** [comment]
Duplicate of #40
